lib/GithubUtils.js

```diff
@@ -0,0 +1,76 @@
+export const GithubOwner = 'Expensify';
```

Comment:
NAB but because these are just constants they should probably be `UPPER_SNAKE_CASE`